apps/alpha-test-app/src/style.css

```diff
@@ -0,0 +1,13 @@
+:root {
```

Comment:
@larsrickert I removed most of the styles that were created by the starter app, should I kick everything except the #app style? what do you think